transcendence-app/src/shared/enums/game-mode.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum GameMode {
```

Comment:
I don't think this should be in the shared folder. Only the game module is concerned about the game modes, I think.

Reply:
You are likely right, I move it to the game folder